src/common.rs

```diff
@@ -337,4 +342,24 @@ mod test {
         );
         assert!(world.read_resource::<Errors>().errors.try_pop().is_none());
     }
+
+    fn test_registrator() {
+        use super::WorldResourceRegistrator;
+        let mut world = World::new();
+
+        pub struct SomeResource {
+            pub v : u32,
+        }
+
+        pub struct TestRegistrator;
+
+        impl WorldResourceRegistrator for TestRegistrator {
+            fn register_with_world(self, world: &mut World) {
+                world.add_resource(SomeResource { v : 12 });
+            }
+        }
+
+        TestRegistrator.register_with_world(&mut world);
```

Comment:
We could add an impl for `World` here which adds an `add_bundle` method. Do you think that's useful @Rhuagh @WaDelma ?